appf.adoc

```diff
@@ -280,13 +280,13 @@ coordinates are identified by the **`standard_name`** attribute value
 **`projection_x_coordinate`** and **`projection_y_coordinate`**
 respectively.
           
-__Notes:__:: Notes on using the **`PROJ.4`** software packages for
-computing the mapping may be found at
-link:$$http://www.remotesensing.org/geotiff/proj_list/geos.html$$[http://www.remotesensing.org/geotiff/proj_list/geos.html].
-These notes assume the point of perspective is directly over the
-equator. A more general description of vertical perspective projection
-is given in <<Snyder>>, pages 169-181.
-           
+__Notes:__:: A general description of vertical perspective projection is
+given in <<Snyder>>, pages 169-181.
+
++
+The corresponding projection is proj.4 is nsper.
```

Comment:
NB. The first use of "is" should probably be "in", but I've honoured exactly what was agreed in the [trac ticket](http://cf-trac.llnl.gov/trac/ticket/71).

Similarly, all other references to proj.4 in this document are in capitals but I've honoured the change proposed in the trac ticket.